code/game/objects/items/weapons/storage/book.dm

```diff
@@ -156,3 +160,42 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 
 /obj/item/weapon/storage/book/bible/booze/PopulateContents()
 	new /obj/item/weapon/reagent_containers/food/drinks/bottle/whiskey(src)
+
+/obj/item/weapon/storage/book/bible/syndicate
+	icon_state ="ebook"
+	deity_name = "The Syndicate"
+	throw_speed = 2
+	throwforce = 18
+	throw_range = 7
+	force = 18
+	hitsound = 'sound/weapons/sear.ogg'
+	damtype = BURN
+	name = "Syndicate Tome"
+	attack_verb = list("attacked", "burned", "blessed", "damned", "scorched")
+	var/ordained = FALSE
+
+/obj/item/weapon/storage/book/bible/syndicate/Initialize()
+	. = ..()
+	
+	
+/obj/item/weapon/storage/book/bible/syndicate/attack_self(mob/living/carbon/human/H)
+	if (!ordained)
+		H.mind.isholy = TRUE
+		ordained = TRUE
+		to_chat(H, "<span class='userdanger'>You try to open the book AND IT BITES YOU!</span>")
+		playsound(src.loc, 'sound/effects/snap.ogg', 50, 1)
+		H.apply_damage(5, BRUTE, pick("l_arm", "r_arm"))
+		to_chat(H, "<span class='notice'>Your name appears on the inside cover, in blood./span>")
+		var/ownername = H.name
+		desc += "<span class='warning'>The name [ownername] is written in blood inside the cover.</span>"
+	return
+
+/obj/item/weapon/storage/book/bible/syndicate/attack(mob/living/M, mob/living/carbon/human/user, heal_mode = TRUE)
+	if (user.a_intent == INTENT_HELP)
+		..()
+	else
+		..(M,user,heal_mode = FALSE)
+	return
```

Comment:
Can you be a bit more specific? I don't get what you're trying to explain to me.  The behavior I am looking for is that if you are on help intent you will try to heal and/or cause brain damage to the target, but if you are on any other intent, then you will attack them.